open-craft/opencraft
# Convert ansible templates to python dictionary format

This PR converts the ansible variable templates to python dictionaries, since previously they were rendered with django templates. This was problematic as both django and Jinja2 templates use the same variable expansion syntax, and both template languages were chained.

**JIRA tickets**: Implements OC-3214

**Testing instructions**:

1. Run unit tests with `make test`
2. Provision new instance and see that templates are rendered properly

**Reviewers**
